packages/sampler/stories/qa/Notification.js

```diff
@@ -0,0 +1,64 @@
+import Notification from '@enact/moonstone/Notification';
+import Popup from '@enact/moonstone/Popup';
+import Button from '@enact/moonstone/Button';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+import {action} from '@storybook/addon-actions';
+
+import {boolean} from '../../src/enact-knobs';
+import {mergeComponentMetadata} from '../../src/utils';
+
+const Config = mergeComponentMetadata('Notification', Notification, Popup);
+
+Notification.displayName = 'Notification';
+
+const filler = '0123456789abcdefghijklmnopqrstuvwxyz';
+
+const messageArray = [filler];
```

Comment:
Curious why you chose to use a module-scoped member here vs instance state. Can you elaborate a bit?

Reply:
I didn't really consider it, it was just how I started with a solution and I left it there :D   I'll move them to instance members, they don't really need to be in the state, I think.